src/index.js

```diff
@@ -0,0 +1,8 @@
+api = require('./api/estados');
+
+const npmPackage = {
+  getCitiesFromState: api.getCities,
+  getStateFromCity: api.getStateFromCity
```

Comment:
Missing trailing comma.

[Read more about it here.](http://eslint.org/docs/rules/comma-dangle)